tests/sles4sap/ase_install.pm

```diff
@@ -0,0 +1,105 @@
+# SUSE's SLES4SAP openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Install SAP ASE via command line using a response file. Both
+# the installation media and the response file are supplied as openQA
+# assets. Verify installation with sles4sap/ase_test.
+#
+# This test module expects the tarball with the installation files for SAP ASE
+# in ASSET_0 and the gzipped response file in ASSET_1
+#
+# Maintainer: QE-SAP <qe-sap@suse.de>
+
+use base 'sles4sap';
+use strict;
+use warnings;
+use testapi;
+use serial_terminal qw(select_serial_terminal);
+use utils qw(file_content_replace);
+use Utils::Logging qw(save_and_upload_log);
+use version_utils qw(is_sle);
+
+=head2 prepare_system_for_ase
+
+  $self->prepare_system_for_ase
+
+Run some preparation commands in the system before SAP ASE's installation.
+This includes setting kernel.randomize_va_space to the recommended value,
+adding the hostname to /etc/hosts, creating a target directory where to
+download the ASE installer and applying the SAP-ASE profile to the
+system via saptune.
+
+=cut
+
+sub prepare_system_for_ase {
+    my ($self, %args) = @_;
+    $args{target} //= '/sapinst';
+    # Add host's IP to /etc/hosts
+    $self->add_hostname_to_hosts;
+    $self->prepare_profile('SAP-ASE');
+    # We need to ensure that the kernel parameter kernel.randomize_va_space is set to 0 in 12-SP3+
+    assert_script_run 'sysctl kernel.randomize_va_space=0' if is_sle('12-SP3+');
```

Comment:
No 12-SP3+ in 2024. We saw in the manual that this is a requirement for 11+ anyway.

Reply:
Noted.